superset/sql_lab.py

```diff
@@ -601,6 +603,23 @@ def execute_sql_statements(
                 serialized_payload = _serialize_payload(
                     payload, cast(bool, results_backend_use_msgpack)
                 )
+
+                # Check the size of the serialized payload
+                if config.get("SQL_LAB_PAYLOAD_MAX_MB"):
+                    serialized_payload_size = sys.getsizeof(serialized_payload)
+                    sql_lab_payload_max_mb = config["SQL_LAB_PAYLOAD_MAX_MB"]
```

Comment:
nit: walrus operator can be used to simplify:
```suggestion
                if sql_lab_payload_max_mb := config.get("SQL_LAB_PAYLOAD_MAX_MB"):
                    serialized_payload_size = sys.getsizeof(serialized_payload)
```